.circleci/config.yml

```diff
@@ -21,7 +21,17 @@ jobs:
           command: |
             git config --global user.email "devhub-deploy@users.noreply.github.com"
```

Comment:
While we're at it, should we update this to match whatever dummy account Alex helped you create?